app/views/admin/facilities/index.html.erb

```diff
@@ -96,15 +64,7 @@
           <div class="row card-row">
             <div class="col-md-4">
               <h5>
-                <% if current_admin.permissions_v2_enabled? %>
-                  <%= link_to facility.name, [:admin, facility_group, facility], class: "d-block" %>
-                <% else %>
-                  <% if policy([:manage, :facility, facility]).show? %>
-                    <%= link_to facility.name, [:admin, facility_group, facility], class: "d-block" %>
-                  <% else %>
-                    <%= facility.name %>
-                  <% end %>
-                <% end %>
+                <%= link_to facility.name, [:admin, facility_group, facility], class: "d-block" %>
```

Comment:
Might be a bug from before - do we not need to show the unhyperlinked names? Does our new scoping ensure we only loop through facilities you have access to, basically?

Reply:
✅